Marker/Classes/Marker.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Marker.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/15/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Returns formatted Markdown text as an attributed string.
+///
+/// - Parameters:
+///   - markdownText: String with Markdown tags.
+///   - textStyle: Text style object containing style information.
+/// - Returns: Formatted Markdown text.
+public func attributedMarkdownString(from markdownText: String,
```

Comment:
I just think that it's slightly nicer from the user side if you can just write `Marker.attributedMarkdownString(...)` instead of `Marker().attributedMarkdownString(...)` just like how, in Alamofire, you can call `Alamofire.request(...)` if you choose to without having to create a `SessionManager`.